doc/configuration.xml

```diff
@@ -46,10 +46,10 @@ $ export NIXPKGS_ALLOW_UNFREE=1
 allowUnfreePredicate = (pkg: ...);
 </programlisting>
 
-    Example to allow flash player only:
+    Example to allow flash player and visual studio code only:
 
 <programlisting>
-allowUnfreePredicate = (pkg: pkgs.lib.hasPrefix "flashplayer-" pkg.name);
+allowUnfreePredicate = with builtins; (pkg: elem (parseDrvName pkg.name).name [ "flashplayer" "vscode" ]);
```

Comment:
Not sure about using lib.hasPrefix:
For example: adding zsh- to the list will also match zsh-completion (I know neither is unfree, just an example). parseDrvName should do the right thing or am I missing something?

@Zimmi48 have a look at the line you removed: `pkgs.lib.hasPrefix`.